consumer.go

```diff
@@ -200,7 +182,9 @@ func (f *consumer) loop() {
 			return
 		case <-ticker.C:
 			updated, err := f.sync(false)
-			f.opt.AfterSync(updated, err)
+			if f.opt.AfterSync != nil {
```

Comment:
I think Go way is to provide noops, but this works as well (just more bug-prone, IMO)